include/server_info.h

```diff
@@ -6,14 +6,19 @@
 #include <memory>
 
 #include "config_parser.h"
+#include "request_handler.h"
 
 struct PathInfo {

```

Comment:
We don't actually need the `PathInfo`  struct because the server doesn't need to store the config or ID for each handler. It only needs to store pointers to each handler. After using `RequestHandler::CreateByName` in `Server::getServerInfo`, call the handler's `Init` function which will store the config in the handler.